governance.md

```diff
@@ -11,52 +11,46 @@ InstructLab is made up of several projects that are defined as codebases and ser
+Project Maintainers are responsible for activities surrounding the development and release of code, the operation of any services that they own, or the tasks needed to execute their project (for example, community management or setting up an event booth). Technical decisions for code reside with the project Maintainers unless there is a decision related to multiple maintainer groups that cannot be resolved by those groups. Those cases can be escalated to the Oversight Committee.
 
-Project Maintainers do not need to be software developers; however they must be substantial contributors. For example, if a repository is for documentation it would be appropriate for a project Maintainer to be an editor or technical writer.
+Project Maintainers do not need to be software developers; however, they must be substantial contributors. For example, if a repository is for documentation it would be appropriate for a project Maintainer to be an editor or technical writer.
 
 Advancement to the project Maintainer position, removal or stepping down, and duties are detailed in the [Contributor Roles](https://github.com/instructlab/community/blob/main/CONTRIBUTOR_ROLES.md). The list of current maintainers can be found [here](https://github.com/instructlab/community/blob/main/MAINTAINERS.md).
 
-## InstructLab Oversight Committee overview
+## InstructLab Oversight Committee Overview
 
-The Oversight Committee is selected by project Maintainers and consists of 3 to 9 leaders on the InstructLab project; these members will serve to supervise the overall project and its health. It will also consist of a selected chair member who will set agendas and call meetings; these meetings can be public or private at the discretion of the Oversight Committee.
+The initial Oversight Committee at the launch of the project was appointed by the founding companies of the InstructLab project. This bootstrap committee will serve until the first election of the Oversight Committee using processes and timing as determined by this group.
+
+The Oversight Committee consists of 3 to 7 leaders on the InstructLab project.  These members will serve to supervise the overall project and its health. It will also consist of a selected Chair member who will set agendas and call meetings. These meetings can be public or private at the discretion of the Oversight Committee.
 
 The Oversight Committee is responsible for the following duties:
 
 * Maintaining the mission, vision, values, and scope of the project
 * Refining the governance and charter as needed
-* Making project level decisions, including setting technical policies that apply across all components
-* Resolving escalated project decisions when the team responsible is blocked
+* Making project-level decisions, including setting technical policies that apply across all components
+* Resolving escalated project decisions when the team responsible is blocked.
```

Comment:
Adding a period here makes this line inconsistent with all others in the list. Please remove it.